src/styles/utils/var.scss

```diff
@@ -0,0 +1,12 @@
+$days_of_week: (
+  'mon': 0,
+  'tue': 1,
+  'wed': 2,
+  'thu': 3,
+  'fri': 4,
+  'sat': 5,
+  'sun': 6,
+);
+$gapSize: 1px;
+$cellSize: 100px;
+$width-calendar: 707px;
```

Comment:
looks like here the possible reason of failed tests.
width of calendar should be 706px (between cells in row 6 gaps)